src/screens/Landing/Navbar/components/package.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
@jerome0caruso Have you created Navbar like it's own package? Why do I see package.json inside Navbar?

Reply:
I just sent that in case.  Yes I ran it as it's own project to try and get the layout completed.

Reply:
I don't think we can have a react app inside a react app. You need to create a Navbar Component inside Components so it can be reused. The reason routing won't work is because now these are two separate projects.